metadata.go

```diff
@@ -139,6 +140,7 @@ type MaterializedViewMetadata struct {
 	MaxIndexInterval        int
 	MemtableFlushPeriodInMs int
 	MinIndexInterval        int
+	ReadRepair              string
```

Comment:
Maybe a small comment here and `ReadRepairChance` to indicate that `ReadRepairChance` was removed in C* 4.0

Reply:
Something like `ReadRepairChance        float64 // Deprecated since Cassandra v4, use ReadRepair instead.`?

Reply:
Maybe something like:

```
ReadRepair              string    // Only present in Cassandra 4.0+
ReadRepairChance        float64   // Note: Cassandra 4.0 removed ReadRepairChance and added ReadRepair instead
```